src/components/alert/style.css

```diff
@@ -1,14 +1,14 @@
 .alert {
-  border-radius: var(--space-3);
-  border-left: var(--space-3) solid;
+  border-radius: var(--eds_spacing_x_small);
   display: inline-block;
-  padding: var(--space-default-gap);
-  padding-left: calc(var(--space-default-gap) - var(--space-3));
+  padding: var(--eds_spacing_medium);
   position: relative;
 }
-
-.alert > svg {
-  margin-right: var(--space-default-gap);
+.alert.icon {
```

Comment:
Create a reusable component for Icon+Content instead for hardcoding how icons should be styled in the Alert.

Reply:
Let's make that a separate task.